src/utils/callHelpers.ts

```diff
@@ -128,54 +126,37 @@ export const soushHarvestBnb = async (sousChefContract, account) => {
     })
 }
 
-const chainId = parseInt(process.env.REACT_APP_CHAIN_ID, 10)
-const cakeBnbPid = 251
-const cakeBnbFarm = farms.find((farm) => farm.pid === cakeBnbPid)
-
-const CAKE_TOKEN = new Token(chainId, getCakeAddress(), 18)
-const WBNB_TOKEN = new Token(chainId, tokens.wbnb.address[chainId], 18)
-const CAKE_BNB_TOKEN = new Token(chainId, getAddress(cakeBnbFarm.lpAddresses), 18)
-
-/**
- * Returns the total CAKE staked in the CAKE-BNB LP
- */
-export const getUserStakeInCakeBnbLp = async (account: string, block?: number) => {
-  try {
-    const masterContract = getMasterchefContract(web3WithArchivedNodeProvider)
-    const cakeBnbContract = getLpContract(getAddress(cakeBnbFarm.lpAddresses), web3WithArchivedNodeProvider)
-    const totalSupplyLP = await cakeBnbContract.methods.totalSupply().call(undefined, block)
-    const reservesLP = await cakeBnbContract.methods.getReserves().call(undefined, block)
-    const cakeBnbBalance = await masterContract.methods.userInfo(cakeBnbPid, account).call(undefined, block)
-
-    const pair: Pair = new Pair(
-      new TokenAmount(CAKE_TOKEN, reservesLP._reserve0.toString()),
-      new TokenAmount(WBNB_TOKEN, reservesLP._reserve1.toString()),
-    )
-    const cakeLPBalance = pair.getLiquidityValue(
-      pair.token0,
-      new TokenAmount(CAKE_BNB_TOKEN, totalSupplyLP.toString()),
-      new TokenAmount(CAKE_BNB_TOKEN, cakeBnbBalance.amount.toString()),
-      false,
-    )
-
-    return new BigNumber(cakeLPBalance.toSignificant(18))
-  } catch (error) {
-    console.error(`CAKE-BNB LP error: ${error}`)
-    return BIG_ZERO
-  }
-}
-
 /**
- * Gets the cake staked in the main pool
+ * Returns the total number of pools that were active at a given block
  */
-export const getUserStakeInCakePool = async (account: string, block?: number) => {
-  try {
-    const masterContract = getMasterchefContract(web3WithArchivedNodeProvider)
-    const response = await masterContract.methods.userInfo(0, account).call(undefined, block)
-
-    return getBalanceAmount(new BigNumber(response.amount))
-  } catch (error) {
-    console.error('Error getting stake in CAKE pool', error)
-    return BIG_ZERO
-  }
+export const getActivePools = async (block?: number) => {
```

Comment:
We send the eligible pools to the voting API so there is a single source of truth. The FE is responsible to let the API know which pools to check for a balance given a block number.